frr-metrics/exporter.go

```diff
@@ -8,6 +8,7 @@ import (
 	stdlog "log"
 	"net/http"
 	"os"
+	"time"
```

Comment:
Why is this required  / raised by gosec?

Reply:
see below, without a ReadTimeout, the defulat is infinite,
which make it easy for denial-of-service (DoS) attacks